docs/getting-started.md

```diff
@@ -6,6 +6,7 @@ However, if you want to integrate containerd into your project we have an easy t
 
 In this guide we will pull and run a redis server with containerd using the client package.
 We will assume that you are running a modern linux host for this example with a compatible build of `runc`.
+You can download one of the latest builds for runc on the [Github release](https://github.com/opencontainers/runc/releases) page and make sure runc installation path registered at `$PATH` variable
```

Comment:
Missing a `.` at the end of the sentence.

Maybe add, "The default install path for `runc` is located at `/usr/local/sbin/runc`."